core/src/main/java/smashdudes/core/input/ControllerInputListener.java

```diff
@@ -10,7 +10,7 @@
  */
 public class ControllerInputListener extends ControllerAdapter implements IGameInputListener
 {
-    private InputState gameInputstate = new InputState();
+    private final InputState gameInputstate = new InputState();
```

Comment:
Won't compile without this change, looks like its from the merge.
```suggestion
    private final InputState gameInputState = new InputState();
```

Reply:
this has been fixed